components/LanguageSidebar.qml

```diff
@@ -0,0 +1,155 @@
+//    used to endorse or promote products derived from this software without specific
+//    prior written permission.
+// 
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+// THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+// THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import "../components" as MoneroComponents
+
+import QtQuick 2.7
+import QtQuick.XmlListModel 2.0
+import QtQuick.Layouts 1.2
+import QtQuick.Controls 2.0
+
+
+Drawer {
+    id: sideBar
+
+    // @TODO: Qt 5.10 introduces `opened` built-in for Drawer
+    property bool isOpened: false
+
+    onClosed: {
+        isOpened = false;
+    }
+
+    onOpened: {
+        isOpened = true;
+    }
+
+    width: 240 * scaleRatio
+    height: parent.height - (persistentSettings.customDecorations ? 50 : 0)
+    y: titleBar.height
+
+    background: Rectangle {
+        color: "#0d0d0d"
+        width: parent.width
+    }
+
+    ColumnLayout {
+        anchors.top: parent.top
+        anchors.left: parent.left
+        anchors.right: parent.right
+        spacing: 0
+
+        ListView {
+            clip: true
+            boundsBehavior: Flickable.StopAtBounds
+            anchors.top: parent.top
+            width: sideBar.width
+            height: sideBar.height
+
+            model: langModel
+
+            delegate: ColumnLayout {
+                width: sideBar.width
+                spacing: 0
+
+                Text {
+                    Layout.topMargin: 8 * scaleRatio
+                    Layout.bottomMargin: 8 * scaleRatio
+                    Layout.leftMargin: 20 * scaleRatio
+                    font.bold: true
+                    font.pixelSize: 14 * scaleRatio
+                    color: MoneroComponents.Style.defaultFontColor
+                    text: display_name
+                }
+
+                Rectangle {
+                    color: MoneroComponents.Style.dividerColor
+                    opacity: MoneroComponents.Style.dividerOpacity
+                    Layout.preferredHeight: 1
+                    Layout.fillWidth: true
+                }
+
+                // button gradient while checked
+                Image {
+                    anchors.fill: parent
+                    source: "../images/menuButtonGradient.png"
+                    opacity: 0.75
+                    visible: true
+
+                    MouseArea {
```

Comment:
A mouseover effect, like that on the accounts page's listview, would be nice. 
i.e.                                 
```
                                onEntered: {
                                    delegateTable.color = "#26FFFFFF"
                                }
                                onExited: {
                                    delegateTable.color = "transparent"
                                }
```

Reply:
The background of those items is actually a picture. However, I was able to simulate a hover effect playing around with `opacity`. Looks better indeed.